### DISCUSSION TITLE
ColliderBuilder::convex_decomposition results in error message

### DISCUSSION THREAD

**Discussion by lairdkruger** [dimforge/rapier#731] [General] [answered]
> Cannot compute exact convex hull without voxel-to-primitives-map. Consider passing voxel_to_primitives_map = true to the voxelizer.

I've not been able to get any convex_decomposition to work with the few basic shapes I've tried (even just a cube)
They all result in the same error, which seems to me to be something from within the API.

Wondering if anyone has been able to get the convex_decomposition methods working recently?

**Comment by ThierryBerger** [comment]
Hello, Sorry to hear that!

Do you have an example code so we can start from that to make progress?

**Comment by lairdkruger** [comment, marked as answer]
Yeah sorry, turns out I was supplying the function with empty vecs. Error message threw me off haha, nothing major.
